models/quality_measures/intermediate/nqf0059_diabetes_hemoglobin_a1c/quality_measures__int_nqf0059_numerator.sql

```diff
@@ -51,10 +54,8 @@ with denominator as (
     , row_number() over(partition by labs.patient_id order by coalesce(collection_date,result_date) desc) as rn
     from labs
     inner join hba1c_test_code
-      on ( labs.normalized_code = hba1c_test_code.code
-       and labs.normalized_code_type = hba1c_test_code.code_system )
-      or ( labs.source_code = hba1c_test_code.code
-       and labs.source_code_type = hba1c_test_code.code_system )
+        on coalesce(labs.normalized_code, labs.source_code) = hba1c_test_code.code
+            and coalesce(labs.normalized_code_type, labs.source_code_type) = hba1c_test_code.code_system
```

Comment:
Individual normalized code and type matching with value set and again source code and type matching with value set has been replaced with coalescing both codes, as for other tables. Please confirm if this works for labs as well.